public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,130 @@
+# list
+
+List <abbr>repository</abbr> contents, including files and directories tracked
+by DVC (<abbr>data artifacts</abbr>) and by Git.
+
+## Synopsis
+
+```usage
+usage: dvc list [-h] [-q | -v] [-R] [--outs-only] [--rev [REV]] url [target]
+
+positional arguments:
+  url              Location of DVC repository to list.
+  target           Path to directory within the repository to list contents for.
+```
+
+## Description
+
+List files, dirs and <abbr>data artifacts</abbr> for the pointed url. The output
+is sorted lexicographically.
+
+The `url` argument specifies the address of the DVC or Git repository containing
+the data source. Both HTTP and SSH protocols are supported for online repos
+(e.g. `[user@]server:project.git`). `url` can also be a local file system path.
+When the url is remote git url the content is checkout into temporary directory.
+
+`--outs-only` option allows to check all <abbr>data artifacts</abbr> into the
+repo, which can be useful before `dvc get` or `dvc import` invocation.
+
+The `target` argument allows to check existense of the target into the repo.
```

Comment:
not sure I understand this to be honest ... what does target actually do?

Reply:
I think this is important enough to make it a second paragraph and explain better what does target actually do